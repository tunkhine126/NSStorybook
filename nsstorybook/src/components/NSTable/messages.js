/*
 * NSTable Messages
 *
 * This contains all the text for the NSTable component.
 */

import { defineMessages } from 'react-intl';

export const scope = 'app.component.NSTable';

export default defineMessages({
  all: {
    id: `${scope}.all`,
    defaultMessage: 'All',
  },
  add: {
    id: `${scope}.add`,
    defaultMessage: 'Add',
  },
  createSurvReq: {
    id: `${scope}.createSurvReq`,
    defaultMessage: 'Create Survey Requests',
  },
  columns: {
    id: `${scope}.columns`,
    defaultMessage: 'Columns',
  },
  customField: {
    id: `${scope}.customField`,
    defaultMessage: 'Custom Field',
  },
  density: {
    id: `${scope}.density`,
    defaultMessage: 'Density',
  },
  export: {
    id: `${scope}.export`,
    defaultMessage: 'Export',
  },
  filters: {
    id: `${scope}.filters`,
    defaultMessage: 'Filters',
  },
  field: {
    id: `${scope}.field`,
    defaultMessage: 'Field',
  },
  itemsSelected: {
    id: `${scope}.itemsSelected`,
    defaultMessage: 'items selected',
  },
  itemSelected: {
    id: `${scope}.itemSelected`,
    defaultMessage: 'item selected',
  },
  items: {
    id: `${scope}.items`,
    defaultMessage: 'items',
  },
  next: {
    id: `${scope}.next`,
    defaultMessage: 'Next',
  },
  of: {
    id: `${scope}.of`,
    defaultMessage: 'of',
  },
  prev: {
    id: `${scope}.prev`,
    defaultMessage: 'Prev',
  },
  requests: {
    id: `${scope}.requests`,
    defaultMessage: 'requests',
  },
  selectAll: {
    id: `${scope}.selectAll`,
    defaultMessage: 'Select all',
  },
  saveChanges: {
    id: `${scope}.saveChanges`,
    defaultMessage: 'Save Changes',
  },
  tableSettings: {
    id: `${scope}.tableSettings`,
    defaultMessage: 'Table Settings',
  },
  reset: {
    id: `${scope}.reset`,
    defaultMessage: 'Reset',
  },
  resultsPerPage: {
    id: `${scope}.resultsPerPage`,
    defaultMessage: 'Results per page',
  },
  reorder: {
    id: `${scope}.reorder`,
    defaultMessage: 'Re-Order...',
  },
  reorderDesc: {
    id: `${scope}.reorderDesc`,
    defaultMessage: 'Drag column label to reorder',
  },
  saveNewView: {
    id: `${scope}.saveNewView`,
    defaultMessage: 'Save New View',
  },
  saveNewViewDesc: {
    id: `${scope}.saveNewViewDesc`,
    defaultMessage:
      'Create a new custom view with your current filter and table settings.',
  },
  newTableView: {
    id: `${scope}.newTableView`,
    defaultMessage: 'New Table View',
  },
  loadTableView: {
    id: `${scope}.loadTableView`,
    defaultMessage: 'Load Table View',
  },
  newViewNotice: {
    id: `${scope}.newViewNotice`,
    defaultMessage:
      'This will create a new custom view with your current filter and table settings.',
  },
  inputPlaceholder: {
    id: `${scope}.inputPlaceholder`,
    defaultMessage: 'Enter a name for this view...',
  },
  noRequests: {
    id: `${scope}.noRequests`,
    defaultMessage: 'No requests found',
  },
});
