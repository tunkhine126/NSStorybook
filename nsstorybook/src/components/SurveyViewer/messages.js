/*
 * SurveyViewer Messages
 *
 * This contains all the text for the SurveyViewer Component.
 */

import { defineMessages } from 'react-intl';

export const scope = 'app.component.SurveyViewer';

export default defineMessages({
  bySurveyor: {
    id: `${scope}.bySurveyor`,
    defaultMessage: 'by Surveyor',
  },
  completed: {
    id: `${scope}.completed`,
    defaultMessage: 'Completed',
  },
  conditionalDisplay: {
    id: `${scope}.conditionalDisplay`,
    defaultMessage: 'Display this question if',
  },
  field: {
    id: `${scope}.field`,
    defaultMessage: 'Field',
  },
  noResponse: {
    id: `${scope}.noResponse`,
    defaultMessage: 'No Response',
  },
  questions: {
    id: `${scope}.questions`,
    defaultMessage: 'Questions',
  },
  question: {
    id: `${scope}.question`,
    defaultMessage: 'Question',
  },
});
