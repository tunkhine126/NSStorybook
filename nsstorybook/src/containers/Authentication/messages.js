/*
 * Authentication Messages
 *
 * This contains all the text for the Authentication container.
 */

import { defineMessages } from 'react-intl';

export const scope = 'app.containers.Authentication';

export default defineMessages({
  header: {
    id: `${scope}.header`,
    defaultMessage: 'Log In',
  },
  logout: {
    id: `${scope}.logout`,
    defaultMessage: `You've successfully logged out!`,
  },
  login: {
    id: `${scope}.login`,
    defaultMessage: `Log In`,
  },
  email: {
    id: `${scope}.email`,
    defaultMessage: `Email*`,
  },
  password: {
    id: `${scope}.password`,
    defaultMessage: `Password*`,
  },
  rememberMe: {
    id: `${scope}.rememberMe`,
    defaultMessage: `Remember Me`,
  },
  forgotPassword: {
    id: `${scope}.forgotPassword`,
    defaultMessage: `Forgot your password?`,
  },
  unlockInstructions: {
    id: `${scope}.unlockInstructions`,
    defaultMessage: `Didn't receive unlock instructions?`,
  },
});
