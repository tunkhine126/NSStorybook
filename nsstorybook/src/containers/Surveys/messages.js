/*
 * Surveys Messages
 *
 * This contains all the text for the Surveys container.
 */

import { defineMessages } from 'react-intl';

export const scope = 'app.containers.Surveys';

export default defineMessages({
  allTime: {
    id: `${scope}.allTime`,
    defaultMessage: 'All Time',
  },
  average: {
    id: `${scope}.average`,
    defaultMessage: 'Average',
  },
  answeredQuestion: {
    id: `${scope}.answeredQuestion`,
    defaultMessage: 'answered this question',
  },
  availableContexts: {
    id: `${scope}.availableContexts`,
    defaultMessage: 'Languages',
  },
  communities: {
    id: `${scope}.communities`,
    defaultMessage: 'All Communities',
  },
  dateRange: {
    id: `${scope}.dateRange`,
    defaultMessage: 'Date Range',
  },
  dataFrom: {
    id: `${scope}.dataFrom`,
    defaultMessage: 'Data From...',
  },
  lastUpdate: {
    id: `${scope}.lastUpdate`,
    defaultMessage: 'Last Update',
  },
  lastSubmission: {
    id: `${scope}.lastSubmission`,
    defaultMessage: 'Last Submission',
  },
  name: {
    id: `${scope}.name`,
    defaultMessage: 'Survey Template',
  },
  median: {
    id: `${scope}.median`,
    defaultMessage: 'Median',
  },
  minimum: {
    id: `${scope}.minimum`,
    defaultMessage: 'Minimum',
  },
  maximum: {
    id: `${scope}.maximum`,
    defaultMessage: 'Maximum',
  },
  outOf: {
    id: `${scope}.outOf`,
    defaultMessage: 'out of',
  },
  questionsCount: {
    id: `${scope}.questionsCount`,
    defaultMessage: 'Questions',
  },
  response: {
    id: `${scope}.response`,
    defaultMessage: 'Response',
  },
  responses: {
    id: `${scope}.responses`,
    defaultMessage: 'Responses',
  },
  submissionsCount: {
    id: `${scope}.submissionsCount`,
    defaultMessage: 'Submissions',
  },
  surveyListHeader: {
    id: `${scope}.surveyListHeader`,
    defaultMessage: 'Survey Templates',
  },
  viewFamily: {
    id: `${scope}.viewFamily`,
    defaultMessage: 'View Family',
  },
  submissionListHeader: {
    id: `${scope}.submissionListHeader`,
    defaultMessage: 'Submissions',
  },
  viewSubmission: {
    id: `${scope}.viewSubmission`,
    defaultMessage: 'View Submissiom',
  },
});
