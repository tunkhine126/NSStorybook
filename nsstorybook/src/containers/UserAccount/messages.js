/*
 * UserAccount Messages
 *
 * This contains all the text for the UserAccount container.
 */

import { defineMessages } from 'react-intl';

export const scope = 'app.containers.UserAccount';

export default defineMessages({
  header: {
    id: `${scope}.header`,
    defaultMessage: 'This is the UserAccount container!',
  },
  accSettingsHeader: {
    id: `${scope}.accSettingsHeader`,
    defaultMessage: 'Account Settings',
  },
  accountUpdated: {
    id: `${scope}.accountUpdated`,
    defaultMessage: 'Account Settings Updated.',
  },
  confirmChanges: {
    id: `${scope}.confirmChanges`,
    defaultMessage: 'Confirm Changes',
  },
  confirmPassword: {
    id: `${scope}.confirmPassword`,
    defaultMessage: 'Confirm Your New Password',
  },
  currentPassword: {
    id: `${scope}.currentPassword`,
    defaultMessage: 'Current Password',
  },
  defaultSurveyLang: {
    id: `${scope}.defaultSurveyLang`,
    defaultMessage: 'Default Survey Language',
  },
  defaultLang: {
    id: `${scope}.defaultLang`,
    defaultMessage: 'Default Language',
  },
  emailChange: {
    id: `${scope}.emailChange`,
    defaultMessage: 'To change your email please contact us.',
  },
  emailLabel: {
    id: `${scope}.emailLabel`,
    defaultMessage: 'Your Email',
  },
  firstNameLabel: {
    id: `${scope}.firstNameLabel`,
    defaultMessage: 'First Name',
  },
  general: {
    id: `${scope}.general`,
    defaultMessage: 'General',
  },
  lastNameLabel: {
    id: `${scope}.lastNameLabel`,
    defaultMessage: 'Last Name',
  },
  langSettings: {
    id: `${scope}.langSettings`,
    defaultMessage: 'Your Language Settings',
  },
  minCharacters: {
    id: `${scope}.minCharacters`,
    defaultMessage: 'Field must be at least 6 characters',
  },
  invalidPassword: {
    id: `${scope}.invalidPassword`,
    defaultMessage: 'Invalid Password, Please Try Again.',
  },
  newPassword: {
    id: `${scope}.newPassword`,
    defaultMessage: 'New Password',
  },
  password: {
    id: `${scope}.password`,
    defaultMessage: 'Password',
  },
  pwdSettingsHeader: {
    id: `${scope}.pwdSettingsHeader`,
    defaultMessage: 'Change Your Password',
  },
  passwordReq: {
    id: `${scope}.passwordReq`,
    defaultMessage:
      'Please make sure your password is strong and 6 characters minimum.',
  },
  passwordChange: {
    id: `${scope}.passwordChange`,
    defaultMessage: 'To change your password please click the link below',
  },
  passwordUpdated: {
    id: `${scope}.passwordUpdated`,
    defaultMessage: 'Password Updated.',
  },
  passwordMatch: {
    id: `${scope}.passwordMatch`,
    defaultMessage: 'Passwords must match.',
  },
  passwordLink: {
    id: `${scope}.passwordLink`,
    defaultMessage: 'Change Your Password',
  },
  organization: {
    id: `${scope}.organization`,
    defaultMessage: 'Organization',
  },
});
