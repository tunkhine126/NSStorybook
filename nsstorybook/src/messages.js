/*
 * General App Messages
 *
 * This contains common app wide text.
 */

import { defineMessages } from 'react-intl';

export const scope = 'app';

export default defineMessages({
  addFilters: {
    id: `${scope}.addFilters`,
    defaultMessage: 'Add Filters...',
  },
  addRecipientChild: {
    id: `${scope}.addRecipientChild`,
    defaultMessage: 'Add Recipient',
  },
  changePhoto: {
    id: `${scope}.changePhoto`,
    defaultMessage: 'Click to change photo',
  },
  cancel: {
    id: `${scope}.cancel`,
    defaultMessage: 'Cancel',
  },
  community: {
    id: `${scope}.community`,
    defaultMessage: 'Community',
  },
  completed: {
    id: `${scope}.completed`,
    defaultMessage: 'Completed',
  },
  closeWindow: {
    id: `${scope}.closeWindow`,
    defaultMessage: 'Close Window',
  },
  create: {
    id: `${scope}.create`,
    defaultMessage: 'Create',
  },
  currentVersion: {
    id: `${scope}.currentVersion`,
    defaultMessage: 'Current Version',
  },
  defaultView: {
    id: `${scope}.rtDashboard`,
    defaultMessage: 'Dashboard',
  },
  deleteRecipient: {
    id: `${scope}.deleteRecipient`,
    defaultMessage: 'Delete Recipient',
  },
  dropFile: {
    id: `${scope}.dropFile`,
    defaultMessage: 'Drop your file here ...',
  },
  export: {
    id: `${scope}.export`,
    defaultMessage: 'Export',
  },
  edit: {
    id: `${scope}.edit`,
    defaultMessage: 'Edit',
  },
  files: {
    id: `${scope}.files`,
    defaultMessage: 'Files',
  },
  fieldHistory: {
    id: `${scope}.fieldHistory`,
    defaultMessage: 'Field History',
  },
  fullProfile: {
    id: `${scope}.fullProfile`,
    defaultMessage: 'Full Profile',
  },
  generalInfo: {
    id: `${scope}.generalInfo`,
    defaultMessage: 'General Info',
  },
  impact: {
    id: `${scope}.impact`,
    defaultMessage: 'Impact',
  },
  languages: {
    id: `${scope}.languages`,
    defaultMessage: 'Languages',
  },
  newFamily: {
    id: `${scope}.newFamily`,
    defaultMessage: 'New Family',
  },
  networkError: {
    id: `${scope}.networkError`,
    defaultMessage: `There seems to be a network issue. Please Try again.`,
  },
  newSurvey: {
    id: `${scope}.newSurvey`,
    defaultMessage: 'New Survey',
  },
  noSurveys: {
    id: `${scope}.noSurveys`,
    defaultMessage: 'No surveys',
  },
  moreBtn: {
    id: `${scope}.moreBtn`,
    defaultMessage: 'More...',
  },
  overview: {
    id: `${scope}.overview`,
    defaultMessage: 'Overview',
  },
  preview: {
    id: `${scope}.preview`,
    defaultMessage: 'Preview',
  },
  profile: {
    id: `${scope}.profile`,
    defaultMessage: 'Profile',
  },
  progressBarDesc: {
    id: `${scope}.progressBarDesc`,
    defaultMessage: 'Progess Bar Description',
  },
  rtDashboard: {
    id: `${scope}.rtDashboard`,
    defaultMessage: 'Dashboard',
  },
  rtSurveys: {
    id: `${scope}.rtSurveys`,
    defaultMessage: 'Surveys',
  },
  rtSubmissions: {
    id: `${scope}.rtSubmissions`,
    defaultMessage: 'Submissions',
  },
  rtRequests: {
    id: `${scope}.rtRequests`,
    defaultMessage: 'Requests',
  },
  rtTemplates: {
    id: `${scope}.rtTemplates`,
    defaultMessage: 'Templates',
  },
  survey: {
    id: `${scope}.survey`,
    defaultMessage: 'Survey',
  },
  rtSurveyTemplates: {
    id: `${scope}.rtSurveyTemplates`,
    defaultMessage: 'Survey Templates',
  },
  rtFamily: {
    id: `${scope}.rtFamily`,
    defaultMessage: 'Family',
  },
  rtFamilies: {
    id: `${scope}.rtFamilies`,
    defaultMessage: 'Families',
  },
  rtRecipientList: {
    id: `${scope}.rtRecipientList`,
    defaultMessage: 'Family List',
  },
  rtReports: {
    id: `${scope}.rtReports`,
    defaultMessage: 'Reports',
  },
  rtRecipients: {
    id: `${scope}.rtRecipients`,
    defaultMessage: 'Recipients',
  },
  rtSettings: {
    id: `${scope}.rtSettings`,
    defaultMessage: 'Settings',
  },
  rtOrg: {
    id: `${scope}.rtOrg`,
    defaultMessage: 'Your Organization',
  },
  rtAccount: {
    id: `${scope}.rtAccount`,
    defaultMessage: 'Your Account',
  },
  rtAccSettings: {
    id: `${scope}.rtAccSettings`,
    defaultMessage: 'Account Settings',
  },
  rtPwdSettings: {
    id: `${scope}.rtPwdSettings`,
    defaultMessage: 'Password Settings',
  },
  rtLogout: {
    id: `${scope}.rtLogout`,
    defaultMessage: 'Log Out',
  },
  recipientID: {
    id: `${scope}.recipientID`,
    defaultMessage: 'Family ID',
  },
  required: {
    id: `${scope}.required`,
    defaultMessage: 'Required.',
  },
  reports: {
    id: `${scope}.reports`,
    defaultMessage: 'Reports',
  },
  requiredField: {
    id: `${scope}.requiredField`,
    defaultMessage: 'Field is required.',
  },
  removeImg: {
    id: `${scope}.removeImg`,
    defaultMessage: 'Remove Image',
  },
  en: {
    id: `${scope}.en`,
    defaultMessage: 'EN',
  },
  es: {
    id: `${scope}.es`,
    defaultMessage: 'ES',
  },
  optionalFields: {
    id: `${scope}.optionalFields`,
    defaultMessage: 'Optional Fields',
  },
  search: {
    id: `${scope}.search`,
    defaultMessage: 'Search...',
  },
  save: {
    id: `${scope}.save`,
    defaultMessage: 'Save',
  },
  snapshot: {
    id: `${scope}.snapshot`,
    defaultMessage: 'Snapshot',
  },
  saved: {
    id: `${scope}.saved`,
    defaultMessage: 'Saved!',
  },
  submissions: {
    id: `${scope}.submissions`,
    defaultMessage: 'Submissions',
  },
  surveyor: {
    id: `${scope}.surveyor`,
    defaultMessage: 'Surveyor',
  },
  reset: {
    id: `${scope}.reset`,
    defaultMessage: 'Reset',
  },
  translations: {
    id: `${scope}.translations`,
    defaultMessage: 'Translations',
  },
  undo: {
    id: `${scope}.undo`,
    defaultMessage: 'Undo',
  },
  update: {
    id: `${scope}.update`,
    defaultMessage: 'Update',
  },
  updateStatus: {
    id: `${scope}.updateStatus`,
    defaultMessage: 'Update Status',
  },
  version: {
    id: `${scope}.version`,
    defaultMessage: 'Version',
  },
  address: {
    en: 'Address',
    es: 'Dirección',
    pt: 'Endereço',
    ht: 'Adrès',
  },
  allFiles: {
    en: 'All Files',
    es: 'Todos los archivos',
    pt: 'Todos os arquivos',
    ht: 'Tout Fichye yo',
  },
  compSurveys: {
    en: 'Completed Surveys',
    es: 'Encuestas completadas',
    pt: 'Pesquisas concluídas',
    ht: 'Sondaj ki Fin Fèt',
  },
  daughter: {
    en: 'Daughter',
    es: 'Hija',
    pt: 'Filha',
    ht: 'Pitit Fi',
  },
  editSurveyRequests: {
    en: 'Edit Survey Requests',
    es: 'Editar solicitudes de encuesta',
    pt: 'Editar solicitações de pesquisa',
    ht: 'Modifye Demann Sondaj yo',
  },
  editFamily: {
    en: 'Edit Family',
    es: 'Editar familia',
    pt: 'Editar Família',
    ht: 'Edit Fanmi',
  },
  lastUpdate: {
    en: 'Last Update',
    es: 'Última actualización',
    pt: 'Última atualização',
    ht: 'Dènye Mizajou',
  },
  familyMemberCreated: {
    en: 'Family Member Created!',
    es: 'Miembro de la familia creado!',
    pt: 'Membro da família criado!',
    ht: 'Manm fanmi kreye!',
  },
  recipientListHeader: {
    en: 'Families',
    es: 'Familias',
    pt: 'Famílias',
    ht: 'Fanmi yo',
  },
  recipientChildren: {
    en: 'Family Members',
    es: 'Miembros de familia',
    pt: 'Membros da família',
    ht: 'Manm Fanmi yo',
  },

  grandmother: {
    en: 'Grandmother',
    es: 'Abuela',
    pt: 'Avó',
    ht: 'Grann',
  },
  grandfather: {
    en: 'Grandfather',
    es: 'Abuelo',
    pt: 'Avô',
    ht: 'Granpapa',
  },
  headOfHousehold: {
    en: 'Head of Household',
    es: 'Jefe de hogar',
    pt: 'Chefe de família',
    ht: 'Chèf Fanmi an',
  },
  inProgress: {
    en: 'In Progress',
    es: 'En progreso',
    pt: 'Em curso',
    ht: 'Ankou',
  },
  lastUpload: {
    en: 'Last Upload',
    es: 'Última carga',
    pt: 'Último upload',
    ht: 'Dènye Telechajman',
  },
  movedIn: {
    en: 'Moved In',
    es: 'Mudado',
    pt: 'Mudou-se',
    ht: 'Antre nan Kay la',
  },
  mobilePhone: {
    en: 'Primary Mobile Phone',
    es: 'Teléfono móvil principal',
    pt: 'Celular principal',
    ht: 'Telefòn Selilè Prensipal',
  },
  pipelineStatus: {
    en: 'Pipeline Status',
    es: 'Estado del canal',
    pt: 'Status do canal',
    ht: 'Stati Transfè',
  },
  surveys: {
    en: 'Surveys',
    es: 'Encuestas',
    pt: 'Questionários',
    ht: 'Sondaj yo',
  },
  surveyRequests: {
    en: 'Survey Requests',
    es: 'Solicitudes de encuesta',
    pt: 'Solicitações de pesquisa',
    ht: 'Demann Sondaj yo',
  },
  surveysCompleted: {
    en: 'Surveys Completed',
    es: 'Encuestas completadas',
    pt: 'Pesquisas concluídas',
    ht: 'Sondaj ki fin Fèt',
  },
  son: {
    en: 'Son',
    es: 'Hijo',
    pt: 'Filho',
    ht: 'Pitit Gason',
  },
  survyRequestBtn: {
    en: 'Survey Request',
    es: 'Solicitud de encuesta',
    pt: 'Solicitação de pesquisa',
    ht: 'Demann Sondaj',
  },
  uploaded: {
    en: 'Uploaded',
    es: 'Cargado',
    pt: 'Carregado',
    ht: 'Sa ki Telechaje',
  },
});
