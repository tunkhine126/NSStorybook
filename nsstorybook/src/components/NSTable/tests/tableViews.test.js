/**
 *
 * Tests for TableViews
 *
 * @see https://github.com/react-boilerplate/react-boilerplate/tree/master/docs/testing
 *
 */

import React from 'react';
import TestingComponent from 'components/shared/Testing';
import { render } from '@testing-library/react';
// import 'jest-dom/extend-expect'; // add some helpful assertions
import TableViews from '../tableViews';
import messages from '../messages';

describe('<TableViews />', () => {
  const options = [
    { name: 'Default', id: 0 },
    { name: 'Ahuachapan Quarterly Audit', id: 1 },
    { name: 'Year-End Report', id: 2 },
  ];
  it('Expect to not log errors in console', () => {
    const spy = jest.spyOn(global.console, 'error');
    render(
      <TestingComponent>
        <TableViews options={options} messages={messages} />
      </TestingComponent>
    );
    expect(spy).not.toHaveBeenCalled();
  });

  it('Should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <TestingComponent>
        <TableViews options={options} messages={messages} />
      </TestingComponent>
    );
    expect(firstChild).toMatchSnapshot();
  });
});
