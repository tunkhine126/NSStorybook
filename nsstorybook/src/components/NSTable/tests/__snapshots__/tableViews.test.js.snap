// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TableViews /> Should render and match the snapshot 1`] = `null`;
